tests/experimental/post_training/test_low_opset_model.py

```diff
@@ -0,0 +1,128 @@
+"""
```

Comment:
Please, place thus test in the following directory - tests/onnx

Reply:
Done